scripts/clone-react-dom-test-suite/index.js

```diff
@@ -0,0 +1,12 @@
+const { clone } = require('./utils');
+
+const repoURL = 'git@github.com:facebook/react.git';
```

Comment:
I'd clone based on tag as well maybe. I don't know that we want to be testing against a master all the time, better to test against the tests for a specific version e.g. 16.2.0

Reply:
I'm worried it'll slow down the cloning. Just clarifying, whats does the workflow look like again. How often will we be running this script?

Reply:
I'd think wed only run it in CI or as one offs probably. I don't think cloning by tag would affect the speed tho (could be wrong tho)

Reply:
Clone just the latest tag?